src/ensembl/src/content/app/entity-viewer/EntityViewer.tsx

```diff
@@ -18,11 +18,12 @@ import {
 import { toggleSidebar } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarActions';
 
 import { StandardAppLayout } from 'src/shared/components/layout';
-import EntityViewerAppBar from 'src/content/app/entity-viewer/components/entity-viewer-app-bar/EntityViewerAppBar';
-import EntityViewerSidebar from './components/entity-viewer-sidebar/EntityViewerSideBar';
-import EntityViewerSidebarTabs from './components/entity-viewer-sidebar-tabs/EntityViewerSidebarTabs';
-import EntityViewerSidebarToolstrip from './components/entity-viewer-sidebar-toolstrip/EntityViewerSidebarToolstrip';
-import EntityViewerTopbar from './components/entity-viewer-topbar/EntityViewerTopbar';
+import GeneView from './gene-view/GeneView';
+import EntityViewerAppBar from './shared/components/entity-viewer-app-bar/EntityViewerAppBar';
+import EntityViewerSidebar from './gene-view/components/entity-viewer-sidebar/EntityViewerSideBar';
+import EntityViewerSidebarTabs from './gene-view/components/entity-viewer-sidebar-tabs/EntityViewerSidebarTabs';
```

Comment:
These would probably have to be renamed (entity-viewer-sidebar -> gene-view-sidebar, etc.); but this can be done at our convenience in a subsequent PR.

Reply:
Renamed `EntityViewerSidebar` and `EntityViewerSidebarTabs` in ad59d2b